lambda/custom/metricsHelper.js

```diff
@@ -16,13 +16,14 @@ class metricsHelper {
 
       const metricsLink = 'https://getpocket.com/v3/send?actions=';
       let actionKeys;
-      if (action == constants.metrics.GET_TITLES) {
+      if (action == constants.metrics.CMD_LISTEN) {
         actionKeys = [
           {
             action: action,
             time: Date.now(),
             cxt_view: 'command',
-            cxt_command: constants.metrics.CTX_CMD_GET_TITLES
+            cxt_command: command,
+            item_id: item_id ? item_id : undefined
```

Comment:
Do you want to provide some string value here like "unknown" rather then undefined? It depends on what they are expecting to receive in the metrics call but `stringify()` will omit the item_id parameter from the object if it's undefined.

Reply:
Or `null` will pass through ok:
```
var obj = {a: 12, b: {x: 34, y:undefined, z:78}, c:null}
JSON.stringify(obj);   //  "{"a":12,"b":{"x":34,"z":78},"c":null}"
```

Reply:
Yes, I wanted to omit it if it's not there so they don't have to deal with the null.  They told me not to send it.

Reply:
👍